src/WebJobs.Script.WebHost/PreJIT/coldstart.jittrace

```diff
@@ -3298,3 +3298,133 @@ int64.ToString(IFormatProvider)~System.Int64,System.Private.CoreLib~1~0~ToString
 string.Concat(ReadOnlySpan`1<char>,ReadOnlySpan`1<char>)~System.String,System.Private.CoreLib~2~0~Concat
 string.Split(char[],int32)~System.String,System.Private.CoreLib~2~0~Split
 string.ToLower()~System.String,System.Private.CoreLib~0~0~ToLower
+[System.Net.Http]System.Net.Http.HttpResponseMessage.get_Content()~System.Net.Http.HttpResponseMessage,System.Net.Http~0~0~get_Content
```

Comment:
How are you adding these? We need to automate adding these because we might need to create a new jittrace file for Windows every now and then and these will get removed.